$OpenBSD: patch-uutil_ml,v 1.1 2005/05/30 19:27:45 sturm Exp $
Post-release developer patch.
o Use aligned writes for better performance (especially on USB devices).
  See http://lists.seas.upenn.edu/pipermail/unison-hackers/2005-May/000090.html
--- uutil.ml.orig	Mon Sep  6 15:15:47 2004
+++ uutil.ml	Fri May 27 16:26:08 2005
@@ -83,7 +83,7 @@ let showProgress i bytes ch =
 (*               Copy bytes from one file_desc to another                    *)
 (*****************************************************************************)
 
-let bufsize = 10000
+let bufsize = 16384
 let bufsizeFS = Filesize.ofInt bufsize
 let buf = String.create bufsize
 
